core/base-service/got.js

```diff
@@ -27,13 +28,6 @@ function requestOptions2GotOptions(options) {
     delete requestOptions.qs
   }
 
-  if ('strictSSL' in requestOptions) {
-    gotOptions.https = {
-      rejectUnauthorized: requestOptions.strictSSL,
-    }
-    delete requestOptions.strictSSL
-  }
-
```

Comment:
if the overall mapping was going to be long-lived i'd wonder if we should keep some kind of check given the object vs. boolean type of the top level key, but given that we're moving away from needing this I suspect that doesn't matter